pyOCD/coresight/ap.py

```diff
@@ -337,8 +337,10 @@ def _handle_error(self, error, num):
 
 class AHB_AP(MEM_AP):
     def init(self, bus_accessible=True):
-        super(AHB_AP, self).init(bus_accessible)
+        # Init super first with bus access disabled, so it doesn't read the ROM table.
+        super(AHB_AP, self).init(False)
```

Comment:
If you move both the setting of auto_increment_page_size and reading the rom table into the MEM_AP class there would be no need to call init twice. This would also more closely match the AP classes themselves. For example if MDM-AP extends AP it will have no concept of a ROM table. What are your thoughts on this?